### ISSUE TITLE
Feature Request - Language list pop up in Wikipedia look up

### ISSUE THREAD

**Issue by yunlhan** [koreader/koreader#4395]
* KOReader version: v2018.12
* Device: Kindle touch 3G (K5)

#### Issue
This post is actually a feature request. 

#### Steps to reproduce

At the moment, if multiple languages are listed, the lookup results will be shown in languages looping over the list of specified languages. For example, if we tell KOReader to query Wikipedia pages specified in `en de fr jp zh` five different languages, first we will be shown search results in English, press the button at bottom center `en > de`, the search results in German come next; the button itself changes to `de > fr`. Press the button again, the search results in French turn up and etc. 

![wikilookup0](https://user-images.githubusercontent.com/6744619/49713443-558f5a80-fc0e-11e8-96b2-b06b9fe0d513.png)

In this example, wouldn’t it be nice if pressing the button `fr > `, a drop down list consisting of four other different languages pops up instead of looping over the entire language list item by item? 

![wikilookup1langlist](https://user-images.githubusercontent.com/6744619/49713530-bc147880-fc0e-11e8-886b-6cac1664e1a0.png)

Thank you.

**Comment by yunlhan** [comment]
@NiLuJe Can I also ask a quick question? I noticed that if I disable WiFi, but leave 3G connection on,  Wkipedia look up will fail. Does that mean Wiki lookup in KOReader only supports WiFi connection? Thanks.

**Comment by NiLuJe** [comment]
I honestly have no idea, but it's quite probable, yes.

**Comment by yunlhan** [comment]
Thanks. 3G connection was a deal breaker should I buy a new kindle. If KOReader does not support 3G Wiki lookup, then I don't think 3G connection matters any longer.

**Comment by poire-z** [comment]
It's quite possible the 3G internet access is limited (https://www.mobileread.com/forums/showthread.php?t=233291), and that network stuff under 3G can only go thru amazon services, including their own wikipedia lookup implementation/proxy'fication.
KOReader actually queries directly the Wikipedia API on Wikipedia servers, with probably woldwide IPs, so Amazon 3G might actually block these connections.

(Never had a Kingle, so that's just speculation :)

**Comment by NiLuJe** [comment]
Yep, Amazon is slowly dropping 3G, (both in terms of software, as @poire-z explained, and in terms of hardware: newer models prefer LTE).

**Comment by yunlhan** [comment]
Thank you @poire-z for the MoRead link. Maybe my next reader will be a Kobo.

@NiLuJe Ah, I didn't mean 3G technology specifically; what I meant was internet access without WiFi as in the name Kindle Touch 3G. So for new devices, whether or nor it has LTE, if KOReader cannot use LTE for wiki lookup, it does not matter to me. In that case, I will consider readers with no LTE.

**Comment by KenMaltby** [comment]
Tethering seems to cover that, although probably not for the spur of the moment lookup.

**Comment by Commodore64user** [comment]
this was fixed by our lovely @poire-z in #14791. round of applause.
